content/articles/2019-01-tibble-2-0-0.Rmarkdown

```diff
@@ -0,0 +1,187 @@
+---
+title: tibble 2.0.0
+slug: tibble-2.0.0
+description: >
+    Tibbles are a modern reimagining of the data frame, keeping what time has shown to be effective, and throwing out what is not, with nicer default output too! This article describes the latest major release and provides an outlook on further developments
+date: 2019-01-06
+author: Kirill Müller
+photo:
+  url: https://unsplash.com/photos/KA89yJKYtjE
+  author: Marcello Gennari
+categories: [package]
+---
+
+
+```{r setup, include = FALSE}
+library(tidyverse)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>", error = TRUE)
+options(rlang_trace_top_env = .GlobalEnv)
+
+options(crayon.enabled = TRUE)
+fansi::set_knit_hooks(knitr::knit_hooks, which = c("output", "error"))
+```
+
+I'm pleased to announce that version 2.0.0 of the *tibble* package is on CRAN now. Tibbles are a modern reimagining of the data frame, keeping what time has shown to be effective, and throwing out what is not, with nicer default output too! Grab the latest version with:
+
+```r
+install.packages("tibble")
+```
+
+This release required a bit of preparation, including a [pre-release blog post](https://www.tidyverse.org/articles/2018/11/tibble-2.0.0-pre-announce/) that described the breaking changes.
+In this blog post, I focus on a few user- and programmer-related changes, and give an outlook over future development:
+
+- 2D columns, `view()`, nameless `enframe()`
+- Robustness, lifecycle, name repair, row names, `glimpse()` for subclasses
+- _vctrs_, smaller footprint, decorations
+
+For a complete overview please see the [release notes](https://github.com/tidyverse/tibble/releases/tag/v2.0.0).
+
+Use the [issue tracker](https://github.com/tidyverse/tibble/issues) to submit bugs or suggest ideas, your contributions are always welcome.
+
+## Changes that affect users
+
+`tibble()` now supports columns that are matrices or data frames.
```

Comment:
Mention that these have always been supported in data frames and are used in some modelling functions, but we're not yet sure how they will be used (but we want to be prepared).

This should probably come last in this section since it's the least impactful to most users.

Reply:
You could also link to https://adv-r.hadley.nz/vectors-chap.html#matrix-and-data-frame-columns